storage/pushsync/pushsync_simulation_test.go

```diff
@@ -186,7 +177,8 @@ func newServiceFunc(ctx *adapters.ServiceContext, bucket *sync.Map) (node.Servic
 	if err != nil {
 		return nil, nil, err
 	}
-
+	// setup netstore
+	netStore := storage.NewNetStore(lstore, enode.HexID(hexutil.Encode(kad.BaseAddr())))
```

Comment:
@nonsense so this was the actual problem, mea culpa.
- pss prox works ok and delivers the chunk in the right places
- but the chunk is synced after only one receipt
- a request can therefore reach a node before push sync reaches it and not asked again until peerskip delay times out.
- the pushsync storer mistakenly used localstore not netstore to put the chunk, therefore no existing fetcher on the node was notified and therefore, the request was only served when the same peer was asked again.

Reply:
```
> a request can therefore reach a node before push sync reaches it and not asked again until peerskip delay times out.
```

How can a request reach a node before push sync, when we are explicitly waiting for push sync receipts before triggering any downloads?

```
> the pushsync storer mistakenly used localstore not netstore to put the chunk, therefore no existing fetcher on the node was notified and therefore, the request was only served when the same peer was asked again.
```

good catch I didn't notice that.